### PR TITLE
Privacy.md

### PR DESCRIPTION
**Pull request by HarryHeights**
This is the privacy policy document I earlier proposed to create. You can review it for corrections and if there are corrections to be made, let me know so I can make the corrections. Thanks.

### CODE REVIEW

File: Privacy.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Information Collection And Use
+
+For better and enhanced utilization, the software is designed to request provision of certain personally identifiable information which is the enablement of several permissions for it to work properly on the device on which the software's services is intended.
+  
+NoteCrypt requires only the enablement of storage permission.
+
+The information that may be collected with the enablement of this permission is stored locally on the device and the developer has no access to that information.
+
+This permission can be revoked when it is not needed, this is privy to only devices running Android 6.0 upward.
+
```
Comment (RyuzakiKK) [inline Privacy.md:16]: Is an `<hr>` missing here?

File: Privacy.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+The security of your data is important, but remember that no method of transmission over the Internet, or method of electronic storage is 100% secure therefore the developer doesn't guarantee its absolute security. <hr>
+
+
+## Children's Privacy
+
+This software is intended for general use and has no restrictions on utilization by children except otherwise stated in a law governing that locale. <hr>
+
+## Links To Other Sites
+
+NoteCrypt does not contain links to any other service provider.The software works independently on it's own.
```
Comment (RyuzakiKK) [inline Privacy.md]: Missing space between provider and the

**Review by RyuzakiKK — CHANGES_REQUESTED** [review]
Awesome, I think this is very good!
I just pointed out a couple of minor issues.

File: Privacy.md (hunk 2 above)
Comment (HarryHeights) [reply Privacy.md]: Alright. I'll fix this

File: Privacy.md (hunk 1 above)
Comment (HarryHeights) [reply Privacy.md:16]: Yes. I missed it. I'll input it now.

**Review by RyuzakiKK — APPROVED** [review]

**Comment by RyuzakiKK** [comment]
Looks good to me, thank you very much.

**Comment by HarryHeights** [comment]
Feels good to contribute. If you need help with any sort of documentation. I'm available. You can send an email to nwosaharry@gmail.com

**Pull request merged**